docker_publisher.py

```diff
@@ -381,6 +381,7 @@ def run():
             'fetchers': {
                 'x86_64': DockerImageFetcherOBS(url="https://build.opensuse.org/public/build/openSUSE:Containers:Leap:15.4/containers/x86_64/opensuse-leap-image:docker", maintenance_release=True),  # noqa: E501
                 'aarch64': DockerImageFetcherOBS(url="https://build.opensuse.org/public/build/openSUSE:Containers:Leap:15.4/containers/aarch64/opensuse-leap-image:docker", maintenance_release=True),  # noqa: E501
+                'armv7l': DockerImageFetcherOBS(url="https://build.opensuse.org/public/build/openSUSE:Containers:Leap:15.4/containers_armv7/armv7l/opensuse-leap-image:docker", maintenance_release=True),  # noqa: E501
```

Comment:
is that armv7 typo vs armv7l intentional?

Reply:
It's intentional - the OBS arch is `armv7l`, the repo is called `containers_armv7`